daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/types/Ledger.scala

```diff
@@ -123,10 +123,11 @@ object Ledger {
       roots: ImmArray[ScenarioNodeId],
       nodes: Map[ScenarioNodeId, Node],
       explicitDisclosure: Relation[ScenarioNodeId, Party],
-      implicitDisclosure: Relation[ScenarioNodeId, Party],
+      localImplicitDisclosure: Relation[ScenarioNodeId, Party],
+      globalImplicitDisclosure: Relation[AbsoluteContractId, Party],
       failedAuthorizations: FailedAuthorizations
   ) {
-    def disclosures = Relation.union(explicitDisclosure, implicitDisclosure)
+    def disclosures = Relation.union(explicitDisclosure, localImplicitDisclosure)
```

Comment:
@dajmaki, @remyhaemmerle-da, @cocreature: This `disclosures` now strictly returns fewer items. Previously `globalImplicitDisclosure` and `localImplicitDisclosure` was conflated when converting `EnrichedTransaction` to `RichTransaction` (assuming that one can easily convert `AbsoluteContractId` to an `ScenarioNodeId`).

This `disclosures` field/method is eventually being used by `ScenarioServiceMain` to convert a ledger to a scenario result to be shown in the IDE. I'm hoping that any bad deviation from the current behavior is caught by some integration tests  between IDE and scenario service. But anyway, could you please confirm that this won't be a problem?